src/goto-checker/report_util.cpp

```diff
@@ -71,6 +71,62 @@ void report_failure(ui_message_handlert &ui_message_handler)
   }
 }
 
+void report_inconclusive(ui_message_handlert &ui_message_handler)
+{
+  messaget msg(ui_message_handler);
+  msg.result() << "INCONCLUSIVE" << messaget::eom;
+
+  switch(ui_message_handler.get_ui())
+  {
+  case ui_message_handlert::uit::PLAIN:
+    break;
+
+  case ui_message_handlert::uit::XML_UI:
+  {
+    xmlt xml("cprover-status");
+    xml.data = "INCONCLUSIVE";
+    msg.result() << xml;
+  }
+  break;
+
+  case ui_message_handlert::uit::JSON_UI:
+  {
+    json_objectt json_result;
+    json_result["cProverStatus"] = json_stringt("inconclusive");
+    msg.result() << json_result;
+  }
+  break;
+  }
+}
+
+void report_error(ui_message_handlert &ui_message_handler)
+{
+  messaget msg(ui_message_handler);
+  msg.result() << "VERIFICATION ERROR" << messaget::eom;
```

Comment:
Applies to all the report* functions: do we print both formatted and unformatted output in case of both XML and JSON?!

Reply:
That's the current behaviour. https://github.com/diffblue/cbmc/commit/387834c492e66aa242a272c7112e6f8abdff928a#diff-6b42a28511965b646db79583668c5e6eL161